faq.md

```diff
@@ -1,143 +1,325 @@
+
+Fundamentally, Flutter is for users that want beautiful apps,
+with delightful motion and animation, and UIs with character
+and an identity all their own.
+
+### What kinds of apps can I build with Flutter?
+
+Flutter is optimized for 2D mobile apps that want to run on
+both Android and iOS.
 
-Flutter has four main components:
+Apps that need to deliver brand-first designs are particularly
+well suited for Flutter. However, apps that need to look like
+stock platform apps can also be built with Flutter.
 
-* a heavily optimized, mobile-first 2D rendering engine (with excellent support
-for text)
-* a functional-reactive framework (optional, you can bring-your-own
-framework)
-* a set of Material Design widgets (optional, you can bring-your-own
-widgets)
-* command-line tools and plugins for [IntelliJ IDEA](https://www.jetbrains.com/idea/),
-including the free Community edition.
+You can build full-featured apps with Flutter, including
+camera, geolocation, network, storage, 3rd-party SDKs, and more.
 
-## What makes Flutter unique?
+### Who uses Flutter?
 
-Flutter is different than most other options for building cross-platform
+Google uses Flutter to build business-critical apps for iOS and Android.
+For example, Google's mobile sales tool app is built with Flutter, along
+with a Store Manager app for Google Shopping Express. Other Flutter
+apps are in the works, or "inside the firewall".
+
+Teams outside of Google are also using Flutter. For example,
+Newsvoice shipped to both
+[iOS](https://itunes.apple.com/nz/app/newsvoice/id1208421834?mt=8) and
+[Android](https://play.google.com/store/apps/details?id=com.newsvoice.newsvoice).
+
+### What makes Flutter unique?
+
+Flutter is different than most other options for building
 mobile apps because Flutter uses neither WebView nor the OEM widgets
 that shipped with the device. Instead, Flutter uses its own high-performance
 rendering engine to draw widgets.
 
-Flutter also offers developers a highly productive and fast development
-experience, fast runtime and engine performance, and beautifully designed
-widgets that make for beautiful apps.
+In addition, Flutter is different because it only has a thin layer of
+C/C++ code. Flutter implements most of its system (compositing, gestures,
+animation, framework, widgets, etc) in _Dart_ (a modern, terse, object-oriented
+language) that developers can
+easily approach read, change, replace, or remove.
+This gives developers tremendous
+control over the system, as well as significantly lowers the bar
+to approachability for the majority of the system.
 
-## Why would I want to invest in learning Flutter?
+### Should I build my next production app with Flutter?
 
-Flutter is an easy way to use a single codebase to deliver beautiful mobile
-apps that run on both Android and iOS. Flutter gives developers quick
-edit/debug cycles for an enjoyable low-friction workflow.
+Flutter is still being developed and is not yet at
+1.0.
```

Comment:
Technically I think react native isn't 1.0 yet either. Should we be less conservative in the language as well?